src/modules/user/routes.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import CreateUserValidation from '../../middlewares/CreateUserValidationSchema';
```

Comment:
Is `CreateUserValidation` a class? If it's not, it should not be named in PascalCase; instead it should be camelCased.